projects/openexr/openexr_deepscanlines_fuzzer.cc

```diff
@@ -108,10 +109,11 @@ static void readFile(T *inpart) {
   }
 }
 
-static void readFileSingle(const char filename[]) {
+static void readFileSingle(IStream& is) {
   DeepScanLineInputFile *file = NULL;
+  Header header(263, 197);
```

Comment:
What is the reason for this new hardcoded header ? Is this expected ?

Reply:
In order to use an `IStream`, I needed to use the `DeepScanLineInputFile (const Header &header, IStream *is, int version, int numThreads = globalThreadCount())` constructor as opposed to the existing one, so a `Header` needed to be instantiated. When looking through examples in the OpenEXR codebase, I found that these numbers were used fairly [frequently](https://github.com/AcademySoftwareFoundation/openexr/search?q=Header+header%28263%2C+197%29%3B&unscoped_q=Header+header%28263%2C+197%29%3B) in testing. I can generate the header width/height using `FuzzedDataProvider` if you'd like, but this may potentially impact any corpora of OpenEXR files we include in the future.

Reply:
corpora is fine to regenerate, arbitrary width and height is more useful for fuzzing, so please make it use FuzzedDataProvider. ping us back when ready for another round.